src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,72 @@
+public final class StringCalculator {
+
+    public String[] enterCommand(String input) throws IllegalArgumentException {
+        if (isBlank(input)) {
+            throw new IllegalArgumentException("Input IllegalArgument Exception");
+        }
+
+        String[] inputs = input.split(" ");
+
+        return inputs;
```

Comment:
```suggestion
        return  input.split(" ");
```
이런 형식으로 변경할 수 있을거 같아요.
로컬 변수를 사용하면 사람은 `이 변수가 무슨 의미였더라?`라고 한번 더 생각하는
시간이 필요하게 되어서 코드 읽기에 시간이 조금 더 필요하게 된답니다.😄
(물론 로컬 변수가 필요한 경우도 있어요🤣)